src/module/settings.ts

```diff
@@ -20,4 +20,14 @@ export const registerSettings = function (): void {
     type: Boolean,
     default: false,
   });
+
+  game.settings.register("swnr", "addShockMessage", {
+    name: "swnr.settings.addShockMessage",
+    hint: "swnr.settings.addShockMessageHint",
+    scope: "world",
+    config: true,
+    type: Boolean,
+    default: true,
+  });
+
```

Comment:
Opposite to the last PR, this should not be an option.
```suggestion
```

Reply:
Sounds like with the discord conversation you were good with this left in as long as the shock was not a separate chat entry. This config will show a roll in the same message for shock if set.